upup/pkg/fi/cloudup/awstasks/route.go

```diff
@@ -45,6 +45,7 @@ type Route struct {
 	InternetGateway           *InternetGateway
 	NatGateway                *NatGateway
 	TransitGatewayID          *string
+	VPCPeeringConnection      *string
```

Comment:
```suggestion
	VPCPeeringConnectionID      *string
```

and all of the other references to this field